src/Client.cpp

```diff
@@ -150,6 +150,13 @@ err_code_t Client::version(types::broadcast_result_t** results, size_t* nHosts)
 }
 
 
+err_code_t Client::quit() {
+  broadcastCommand(keywords::kQUIT, 4);
```

Comment:
I would prefer not to call `strlen` for a **static const variable** at runtime. May be it can be optimised at compile time but I'm not sure.